guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,254 @@
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-pwm.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-pwm@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-pwm-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-pwm-mobile@2x.png"
+   alt-text = "A diagram showing the sequence of recovery scenarios."
+   width = 1600
+   height = 800
+%}
+
+#### Privacy considerations
+This approach makes a potential trade-off in terms of privacy, because the children could access the backup material and recover the wallet. This is acceptable for Alice and Bob, since they have good relationships and trust their children. After all, this is the reason they have chosen to include them in the setup in the first place.
+
+But even if Christina and David would breach their parents’ trust and import the wallet, they would not be able to move any funds, because their inheritance keys are time locked. And since Alice and Bob access their safe regularly, they would notice that the bag with the backup material has been opened. They can then safely move the funds to a new wallet, for which they would implement more restrictive security measures.
+
+### Secondary Access (Assisted Recovery)
+
+##### Separation of Backup Material
+Alice and Bob provide their lawyer, Edward, with an encrypted copy of the recovery kit. They store the password separately in a bank deposit box. This way, Christina and David can access the funds through legal channels if necessary, without compromising privacy.
+
+- **Edward's Office**: Encrypted copy of the recovery kit (without password) in a tamper-evident bag
+- **Bank Deposit Box**: Recovery kit password in a tamper-evident bag
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-secondary.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-secondary@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-secondary-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-secondary-mobile@2x.png"
+   alt-text = "A diagram showing the sequence of recovery scenarios."
+   width = 751
+   height = 397
+%}
+
+
+##### Shared Password Manager
+To make sure that Christina and David will be able to retrieve both the recovery kit  and the password to decrypt it, Alice and Bob store the following pieces of information in a shared family vault:
+
+- House safe PIN
+- Lawyer's contact details
+- Bank deposit box details
+
+### Inheritance Planning
+Alice and Bob keep a copy of their Last Will in the home safe and give another to Edward, ensuring smooth inheritance processes.
```

Comment:
Maybe worth including that the will should mention the bitcoin wallet and refer to the recovery documents.